### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by baileyjrob**
Regular Update

### CODE REVIEW

**Review by alatham21 — APPROVED** [review]
Looks good

**Review by alatham21 — CHANGES_REQUESTED** [review]
looks like this is failing tests. Was this fixed in your other PR?

**Comment by baileyjrob** [comment]
> looks like this is failing tests. Was this fixed in your other PR?

Not yet. My PR just added the temporary fix for the hour checking. Cameron was going to fix the bug, I believe, but I think there was an issue with his fix. I'm not sure if he's addressed it yet.

**Comment by CameronBourque** [comment]
> > looks like this is failing tests. Was this fixed in your other PR?
> 
> Not yet. My PR just added the temporary fix for the hour checking. Cameron was going to fix the bug, I believe, but I think there was an issue with his fix. I'm not sure if he's addressed it yet.

Originally I thought this was a problem with my pull request because of some lines in the schema.db that I deleted. Turns out those lines didn't matter at all. Currently, I don't know what the cause of the problem is and I'm having difficulties figuring that out. Mainly because I can't recreate the issue.

File: app/views/tutoring_session_mailer/link_action_email.text.erb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Your application to the tutoring session scheduled for <%= @tsession.scheduled_datetime.to_date.to_formatted_s(:long_ordinal) %> at <%= @tsession.scheduled_datetime.strftime("%I:%M %p")  %> has been <%= @link_action %>
+==============
+
+The tutor included the following message:
+<%= @message %>
+
+<% if @link_action.downcase == 'denied' %>
+  The application being denied could have been done for a variety of reasons, from being too full or to the session being canceled.
+  If you think this is an error, you can contact the tutor at <%= @to.email %>.
```
Comment (alatham21) [inline app/views/tutoring_session_mailer/link_action_email.text.erb:9]: great addition!

**Review by alatham21 — APPROVED** [review]
Great additions. The email content was very informative.

**Pull request merged**